R/drive_upload.R

```diff
@@ -179,17 +176,17 @@ build_drive_upload <- function(input = NULL,
     id <- proc_res$id
   }
 
-  url <- file.path(.drive$base_url,
-                   "upload/drive/v3/files",
-                   paste0(id, "?uploadType=media"))
-
-  list(
-    method = "PATCH",
-    url = url,
+  build_request(
+    path = "/upload/drive/v3/files/{fileId}",
```

Comment:
I've gotten the impression that `path` is now looked up based on endpoint. Is something different here on purpose?